src/css/index.css

```diff
@@ -0,0 +1,16 @@
+body{
+  font-family: sans-serif;
+}
+
+li{
+  margin: 5px 0;
+}
+
+a{
+  font-size: 18px;
+}
+
+#gallery-examples{
```

Comment:
Why this style is here? shouldn't it be in base.css?
and never add styling on id,
Read more on css specificity: https://medium.com/@emmabostian/css-specificity-d5fdb0996c81

Reply:
It is here because, it is the styling to the different Gallery examples index file